Sample/SextantSample.Avalonia/Views/GreenView.xaml.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reactive.Disposables;
+using Avalonia.Markup.Xaml;
+using Avalonia.ReactiveUI;
+using ReactiveUI;
+using SextantSample.ViewModels;
+
+namespace SextantSample.Avalonia.Views
+{
+    public class GreenView : ReactiveUserControl<GreenViewModel>
+    {
+        public GreenView()
+        {
+            AvaloniaXamlLoader.Load(this);
+            this.WhenActivated(disposables =>
+            {
+                this.WhenAnyValue(x => x.ViewModel)
+                    .BindTo(this, x => x.DataContext)
+                    .DisposeWith(disposables);
+            });
```

Comment:
This weird code in `ReactiveUserControl`s is required to keep the `ViewModel` property in sync with `DataContext`. Currently, AvaloniaUI [listens to `DataContextChanged` and updates the view model](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.ReactiveUI/ReactiveUserControl.cs#L23), but not vice versa. Going to think about how to better resolve this on the Avalonia side.

Reply:
https://github.com/AvaloniaUI/Avalonia/pull/4917 is now merged, so hopefully, we'll be able to use a preview build soon and refactor this.